src/main/java/com/gd/clinic/security/jwt/TokenUtils.java

```diff
@@ -0,0 +1,58 @@
+package com.gd.clinic.security.jwt;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.JwtException;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import io.jsonwebtoken.security.Keys;
+import io.swagger.models.auth.In;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.userdetails.UserDetails;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+
+public class TokenUtils {
+
+
+    private final static String ACCESS_TOKEN_SECRET = "eyJhbGciOiJIUzI1NiIsInR5cCI6Ikp9";
+    private final static Long ACCESS_TOKEN_VALIDITY_SECONDS = 2_592_000L;
```

Comment:
for such value we can use `int`
and are you sure that it's really in seconds?

Reply:
I did some manual test with smaller seconds and it does last with the number that is declared.  In this case 2_592_000L is 2,592,000 seconds which is equivalente to 30 days